site/_posts/2021-02-19-octant-0.17-release.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Kubernetes: there is an app for that"
+image: /img/posts/2020/11/19/survey-results-banner.png
+excerpt: Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems
+author_name: Milan Klanjsek
+author_avatar: /img/contributors/milan-klanjsek.jpg
+categories: ['kubernetes']
+tags: ['octant', 'release']
+---
+
+<h1>Kubernetes: there is an app for that</h1>
+
+Turning Octant into a desktop application has long been a goal of ours and we are pleased to announce that with the release of 0.17 we are shipping a desktop application for all three major operating systems. The command-line version will still be available for the next couple of releases to allow a smooth transition for everyone,after which we will be 100% focused on the desktop version.
+
+![Octant Electron applications view](/img/posts/2021/02/19/electron-applications.png)
+
+
+To create a desktop version of Octant we used Electron. Electron is a popular cross-platform framework used by many well known desktop applications. Electron is a mature technology built on top of Chromium engine with a solid production environment and great community support. Leveling up to a desktop application opens the doors for many new and exciting things Octant can leverage in the future to improve the overall user experience.
+
+Accessing kubernetes clusters from an application brings many benefits, especially access to the hardware and operating system components. One of the benefits we took advantage right away is access to the System tray. Octant can run in the background and be accessible with just a single click from the System tray. If you want to see the logs, you can get to them right there from the System tray. Expect more actions available from the System tray in the near future.
```

Comment:
The first sentence can be removed `Accessing kubernetes clusters ... system components`.

Second sentence: remove `we took advantage right  away`.